peregrine/analysis/print_res.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
Can we also change the name of this file to make it clear that this is to print tracking results? i.e. `print_track_res.py`.